packages/arb-token-bridge-ui/src/components/common/atoms/Switch.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+  Standardized Toggle-switch 
+*/
```

Comment:
nit: do we need these comments on top of files? IMO I don't feel they add much value and they add clutter